### PR TITLE
Change fake-cluster-id to a real uuid

### PR DESCRIPTION
**Pull request by twiest**
- [x] Change fake-cluster-id to a real uuid
- [x] Test the change to ensure it is working correctly

/assign @joelddiaz 
/cc @suhanime

### CODE REVIEW

**Comment by twiest** [comment]
Fixes: https://issues.redhat.com/browse/HIVE-1354

**Review by joelddiaz — COMMENTED** [review]
/lgtm

File: pkg/installmanager/fake.go [hunk 1]
```diff
@@ -4,12 +4,13 @@ import (
...
 	"fmt"
 
+	"github.com/google/uuid"
 	installertypes "github.com/openshift/installer/pkg/types"
 
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 )
 
-const fakeMetadataFormatStr = `{"aws":{"identifier":[{"kubernetes.io/cluster/fake-infraid":"owned"},{"openshiftClusterID":"fake-cluster-id"}],"region":"us-east-1"},"clusterID":"fake-cluster-id","clusterName":"%s","infraID":"fake-infra-id"}`
+const fakeMetadataFormatStr = `{"aws":{"identifier":[{"kubernetes.io/cluster/fake-infraid":"owned"},{"openshiftClusterID":"%s"}],"region":"us-east-1"},"clusterID":"%s","clusterName":"%s","infraID":"fake-infra-id"}`
```
Comment (abhinavdahiya) [inline pkg/installmanager/fake.go:13]: I think the goal of using `fake-cluster-id` was to make it sure clear that this is a fake cluster, replacing this with UUID only now makes these clusters look very similar to real ones which can cause problems.. any reason why we didn't go for the original recommendation from slack thread of `fake-cluster-UUID` @twiest

Comment (twiest) [reply pkg/installmanager/fake.go:13]: For posterity, this pattern was discussed on slack, but didn't make it to the card. Working on a separate PR to change to using this pattern.

Comment (twiest) [reply pkg/installmanager/fake.go:13]: Fixed in PR #1273

**Pull request merged**